### ISSUE TITLE
Content stage only showing `Draft` & not `Published`

### ISSUE THREAD

**Issue by SanyamPunia** [hygraph/hygraph-examples#266]
I've published two documents on the CMS and I'm fetching + rendering them inside my Next.js application.

I want to render only those documents which are in the `published` stage, and not in `draft`.

When I fetch the content using GraphQL in the API playground, the `stage` attribute is always in the `draft`  stage, even after I published it.

---

*GraphQL query*
```graphql
query getBlogStage{
  portfolioBlogs {
    id
    title
    stage
  }
}
```

*Output*

![image](https://user-images.githubusercontent.com/35108041/232670913-9d4cdaaf-1c06-4e9a-90bf-20fbbc164fa6.png)

In the specific content, it's also showing the `published` stage

![image](https://user-images.githubusercontent.com/35108041/232671840-62fc1ac1-3071-4ea5-895b-99cd19c8e36a.png)

I would appreciate if anyone could find possible fix for this.

**Comment by TontofourK** [comment]
Yeah, I am also stuck with the same issue. Please share a possible fix
